src/styles/styles.css

```diff
@@ -441,3 +447,239 @@ footer {
     padding: 0.5rem 0;
   }
 }
+
+.popup-header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  width: 100%;
+  margin: 0 auto;
+  list-style-type: none;
+  padding-top: 0.6rem;
+  color: aliceblue;
+  font-size: 1.5rem;
+}
+
+#headForm {
+  text-align: center;
+  font-size: 1.5rem;
+  font-weight: 800;
+  color: var(--clr-Pop-text);
+}
+
+.form-inline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 0.5rem;
+  margin-top: 3rem;
+  padding: 0.5rem 0.2rem 0.2rem 0.5rem;
+  background-color: rgba(255, 255, 255, 0.6);
+}
+
+.form-inline input {
+  width: 100%;
+  height: 2rem;
+  border: 1px solid #ddd;
+  border-radius: 1rem;
+  padding: 0.5rem;
+  font-size: 1.2rem;
+}
+
+.form-inline label {
+  font-size: 1.2rem;
+  color: var(--clr-dark);
+  font-weight: 600;
+}
+
+.form-inline button {
+  width: 100%;
+  height: 3rem;
+  border: 1px solid #ddd;
+  border-radius: 1rem;
+  padding: 0.5rem;
+  font-size: 1.2rem;
+  background: rgb(255, 0, 72);
+  color: aliceblue;
+  cursor: pointer;
+  transition: all 400ms ease-in-out;
+}
+
+.form-inline button:hover,
+:focus {
+  color: rgb(255, 0, 72);
+  background-color: white;
+}
+
+.hideMe {
+  font-size: 0.4px;
+  opacity: 0;
+}
+
+.popup-Images {
+  width: auto;
+  height: 400px;
+  border-radius: 5px;
+  margin-top: 150px;
+  margin-left: 24px;
+  outline: 5px solid var(--clr-Pop-text);
+}
+
+.popup-Tags {
+  font-size: 1.5rem;
+  font-weight: 600;
+}
+
+.popup-body {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 0.5rem;
+  margin-top: 1rem;
+  padding: 0 0 0.5rem;
+  outline: 1px solid #ddd;
+  outline-style: dashed;
+  background-color: rgba(255, 255, 255, 0.6);
+}
+
+.card-tags {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.successMessage {
+  color: green;
+  font-size: 0.8rem;
+  font-weight: 600;
+  background-color: rgba(0, 250, 0, 0.795);
+  border-radius: 1rem;
+  padding: 0.5rem;
+  margin-top: 1rem;
+}
+
+.popUp {
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 0;
+  background-color: var(--clr-popBG);
+}
+
+.popDet {
+  font-size: 1.5rem;
+  font-weight: 600;
+}
+
+.popup-inner {
+  background-color: var(--clr-popBG) !important;
```

Comment:
Please avoid using too much `!important`, as It can mess up and interfere with your styling. Thanks